analyzer/src/main/kotlin/managers/PIP.kt

```diff
@@ -385,7 +398,22 @@ class PIP(analyzerConfig: AnalyzerConfiguration, repoConfig: RepositoryConfigura
         // Create an out-of-tree virtualenv.
         log.info { "Creating a virtualenv for the '${workingDir.name}' project directory..." }
         val virtualEnvDir = createTempDir(workingDir.name.padEnd(3, '_'), "virtualenv")
-        ProcessCapture(workingDir, "virtualenv", virtualEnvDir.path).requireSuccess()
+
+        //Create virtual environment with correct pip version. Basically we don't know where Pythons were installed.
+        //By default on Linux commands "python" and "python2" are using for Python version 2. After install Python version >3, should be active also command "python3".
+        //On Windows we have to add Python interpreters to environment variables and they should have another names.
+        //See: https://stackoverflow.com/a/22626734/5877109
+        val projectHasPython3 = ProjectVersion.isPython3(workingDir)
+        var virtualEnv = when (Pair(projectHasPython3, OS.isWindows)){
+            Pair(true, true) -> ProcessCapture(workingDir, "virtualenv", virtualEnvDir.path, "-p", "python.exe")
+            Pair(false, true) -> ProcessCapture(workingDir, "virtualenv", virtualEnvDir.path, "-p", "python2.exe")
```

Comment:
At least on my Windows machine actually both Python 2 and Python 3 executables are just called `python.exe` 😞 Also see https://docs.python.org/2/using/windows.html and https://docs.python.org/3/using/windows.html. Our Windows CI running on [AppVeyor does not seem to be different](https://www.appveyor.com/docs/windows-images-software/#python) in that regard, so I'm wondering where `python2.exe` comes from. Can you please elaborate?

Reply:
You have right. I fixed this. Now if you want to start a Python with specific version, you have to use command `py` , e.g. `py -2` or `py -3` 
This should work correctly on every Windows:
https://docs.python.org/3/installing/#work-with-multiple-versions-of-python-installed-in-parallel